src/lucky/route_handler.cr

```diff
@@ -1,15 +1,36 @@
-require "colorize"
-
 class Lucky::RouteHandler
   include HTTP::Handler
 
+  Habitat.create do
+    setting mime_extensions : Hash(String, String) = {} of String => String
+  end
+
   def call(context)
+    original_path = context.request.path
+    format = get_extension(context.request.path)
+    context.request.path = context.request.path.gsub(format.to_s, "") if format
```

Comment:
The gain makes me nervous because it might overwrite query params or other parts of the path as well. Could you add a spec for that to make sure that doesn’t happen?